charts/openfga/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: openfga
 description: A Kubernetes Helm chart for the OpenFGA project.
 
 type: application
-version: 0.2.42
-appVersion: "v1.9.5"
+version: 0.2.43
+appVersion: "v1.9.6"
```

Comment:
Shouldn't this be `v1.10.1`?

https://github.com/openfga/openfga/releases/tag/v1.10.1